### DISCUSSION TITLE
Sequence Redirection on Mobile Issue

### DISCUSSION THREAD

**Discussion by YapCaiFu** [WalletConnect/walletconnect-monorepo#4412] [Developer Support] [answered]
const { open } = useWeb3Modal()
When I used localhost to try the open() in mobile device, and select Sequence Wallet, the own ReactJs website will be auto replaced by sequence app website (https://sequence.app/connect) for login authentication, and there is no way to return back to own website after login successful. 

I am the beginner on using this, any advice on how to control this? Thanks in advance.
![ewallet](https://github.com/WalletConnect/walletconnect-monorepo/assets/26355257/2a9b4c64-d509-403a-a198-707f326971e6)
![ewallet2](https://github.com/WalletConnect/walletconnect-monorepo/assets/26355257/3d083484-b50f-4e30-ae1d-bdf63c2b7aa9)
![ewallet3](https://github.com/WalletConnect/walletconnect-monorepo/assets/26355257/61d06166-c978-4f7d-a079-c21a19504bd1)

**Comment by glitch-txs** [comment]
Hi, thanks for reporting. I'm not able to reproduce the issue.

Could you try testing it here and also sharing the steps to reproduce it? Thanks!
https://web3modal.com/

**Reply by YapCaiFu** [reply]
Hi, here are the steps to reproduce.

1. select the sequence wallet
2. It auto redirects to login
3. After login successfully, the website remain in sequence wallet instead of returning [web3modal.com](https://web3modal.com/)

May I know is there any way to redirect back to the original website after login successful?

Here below is the screenshot, thanks!
![ewallet4](https://github.com/WalletConnect/walletconnect-monorepo/assets/26355257/bbde6437-3aba-4bb1-a40a-77743e7aad19)
![ewallet5](https://github.com/WalletConnect/walletconnect-monorepo/assets/26355257/802fa4eb-8b44-44fe-ae0b-aade9e21671f)
![ewallet6](https://github.com/WalletConnect/walletconnect-monorepo/assets/26355257/cf2d9d4f-4a33-4cd7-a61f-6ebbceee8da0)
![ewallet7](https://github.com/WalletConnect/walletconnect-monorepo/assets/26355257/d49b990f-d38a-41a4-99b9-09dd259671fb)

**Reply by glitch-txs** [reply, marked as answer]
This is not possible for webapps and the user needs to do it manually.
